docs/source/tips_tricks.rst

```diff
@@ -15,12 +15,12 @@ The USB cable you choose can make a big difference in what you see when using yo
 
 Screenshot before and after changing to a noise reducing cable (`view full size image <http://i.imgur.com/e64LASK.jpg>`__): 
 
-.. image:: ../images/687474703a2f2f692e696d6775722e636f6d2f6536344c41534b2e6a7067.jpeg
+.. image:: ../images/noisereducingcablescreenshot.jpeg
 	:align: center
 
 The `cable <http://www.amazon.com/gp/product/B00A99BR90/ref=oh_aui_detailpage_o02_s00?ie=UTF8&psc=1>`__ used when making the screenshot for the difference in noise is: 
 
-.. image:: ../images/687474703a2f2f6563782e696d616765732d616d617a6f6e2e636f6d2f696d616765732f492f34314943364c543361434c2e6a7067.jpeg
+.. image:: ../images/noisereducingcable.jpeg
```

Comment:
Please remove this image (and the file) and the amazon link, changing the sentence to "A shielded cable with ferrite core was used in the right-hand image."

Reply:
Done.